Documentation/async-disaster-recovery.md

```diff
@@ -0,0 +1,460 @@
+# Failover and Failback in RBD Async Disaster Recovery
+
+[RBD mirroring](https://docs.ceph.com/en/latest/rbd/rbd-mirroring/)
+ is an asynchronous replication of RBD images between multiple Ceph clusters.
+ This capability is available in two modes:
+
+* Journal-based: Every write to the RBD image is first recorded
+ to the associated journal before modifying the actual image.
+ The remote cluster will read from this associated journal and
+ replay the updates to its local image.
+* Snapshot-based: This mode uses periodically scheduled or
+ manually created RBD image mirror-snapshots to replicate
+ crash-consistent RBD images between clusters.
+
+## Create RBD Pools
+
+In this section we create specific rbd pools that are RBD mirroring
+ enabled for use with the DR use case.
+
+> :memo: **Note:** It is also feasible to edit existing pools and
+> enable them for replication.
+
+Execute the following steps on each peer cluster to create mirror
+ enabled pools:
+
+* Create a RBD pool that is mirroring enabled by adding the section
+ spec.mirroring in the CephBlockPool CR, example:
+
+    ```yaml
+    apiVersion: ceph.rook.io/v1
+    kind: CephBlockPool
+    metadata:
+      name: replicapool
+      namespace: rook-ceph
+    spec:
+      replicated:
+        size: 1
+      mirroring:
+        enabled: true
+        mode: image
+        # schedule(s) of snapshot
+        snapshotSchedules:
+          - interval: 24h # daily snapshots
+            startTime: 14:00:00-05:00
+    ```
+
+* Creating `replicapool` pool on the cluster:
+
+    ```bash
+    kubectl apply -f pool.yaml -n rook-ceph
+    ```
+
+   > ```bash
+   > cephblockpool.ceph.rook.io/replicapool created
+   > ```
+
+* Repeat the steps on the peer cluster.
+
+> :memo: WARNING: Pool name across the cluster peers should be the same
+> for RBD replication to function.
+
+For more information on CephBlockPool CR, please refer to the [ceph-pool-crd documentation](Documentation/ceph-pool-crd.md#mirroring).
+
+## Bootstrap Peers
+
+In order for the rbd-mirror daemon to discover its peer cluster, the
+ peer must be registered and a user account must be created. The following
+ steps enable bootstrapping peers to discover and authenticate to each other.
+
+For more details, refer to the official rbd mirror documentation on
+ [how to create a bootstrap peer](https://docs.ceph.com/en/latest/rbd/rbd-mirroring/#bootstrap-peers).
+
+## Create RBDMirror CRD
+
+Replication is handled by the rbd-mirror daemon. The rbd-mirror daemon
+ is responsible for pulling image updates from the remote, peer cluster,
+ and applying them to image within the local cluster.
+
+Creation of the rbd-mirror daemon(s) is done through the custom resource
+ definitions (CRDs).
+ For more information on how to set up Ceph RBDMirror CRD, refer to
+ [rook documentation](https://rook.io/docs/rook/master/ceph-rbd-mirror-crd.html).
+
+## Enable OMAP Generator and Volume Replication
+
+### OMap Generator
```

Comment:
IMO this doc does not need to explain details about the sidecars. We just need to tell them to enable the sidecars. So I think you could delete lines 88 - 111, and just focus on the operator settings they need to enable.